azure/azure-explanation/confidential-computing.rst

```diff
@@ -3,16 +3,16 @@ Security features on Azure
 
 Azure provides two types of security features:
 
-* `Trusted launch <https://learn.microsoft.com/en-us/azure/virtual-machines/trusted-launch>`_ which is a set of features including virtual Trusted Platform Module (vTPM) and `secure boot <https://wiki.ubuntu.com/UEFI/SecureBoot>`_
+* `Trusted Launch <https://learn.microsoft.com/en-us/azure/virtual-machines/trusted-launch>`_ which is a set of features including virtual Trusted Platform Module (vTPM) and `secure boot <https://wiki.ubuntu.com/UEFI/SecureBoot>`_
 * `Confidential virtual machine <https://learn.microsoft.com/en-us/azure/confidential-computing/confidential-vm-overview>`_ with support for `AMD Secure Encrypted Virtualisation-Secure Nested Paging (SEV-SNP) <https://www.amd.com/system/files/TechDocs/SEV-SNP-strengthening-vm-isolation-with-integrity-protection-and-more.pdf>`_, along with `measured boot <https://learn.microsoft.com/en-us/azure/security/fundamentals/measured-boot-host-attestation>`_ using a vTPM
 
+See :ref:`Launch Ubuntu Images on Azure` on how to launch Ubuntu images with Trusted Launch and
+Confidential Virtual Machine on Azure.
+
 Trusted launch
 --------------
 
 All Ubuntu images from 20.04 (Focal Fossa) support trusted launch and secure boot on Hyper-V Gen2 instances. 
-To start an Ubuntu instance with vTPM and secure boot enabled, use the following flags from the Azure CLI::
-        
-   --security-type TrustedLaunch --enable-secure-boot true --enable-vtpm
```

Comment:
The remaining two flags (--enable-secure-boot and --enable-vtpm) are not mentioned in the new how-to guide. Are they enabled by default when security type is set to TrustedLaunch?

Reply:
I commented on this in the commit message:

```
Also
refined the Azure CLI commands as it applies to Trusted Launch (e.g.
selecting the TrustedLaunch security type will automatically enable
vTPM).
```

Reply:
Oops sorry, I missed that!